olp-cpp-sdk-core/src/context/ContextInternal.h

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2023 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#pragma once
+
+#include <olp/core/context/Context.h>
+
+#include <memory>
+#include <vector>
+#if defined(__APPLE__)
+#include <list>
+#include <map>
+#include <string>
+#endif
+
+#ifdef ANDROID
+#include <jni.h>
+#elif defined(__APPLE__)
+#include <olp/core/context/EnterBackgroundSubscriber.h>
+#endif
+
+namespace olp {
+namespace context {
+
+// Because of static initializer orders, we need to have this on the heap.
+struct ContextData {
+  std::vector<olp::context::Context::InitializedCallback> initVector;
+  std::vector<olp::context::Context::DeinitializedCallback> deinitVector;
+
+  std::mutex contextMutex;
+  size_t contextInstanceCounter = 0;
+#ifdef ANDROID
+  JavaVM* javaVM = nullptr;
+  jobject context = nullptr;
+#elif defined(__APPLE__)
+  std::list<std::weak_ptr<EnterBackgroundSubscriber> >
+      enterBackgroundSubscibers;
+  std::map<std::string, std::function<void()> > completionHandlers;
+#endif
```

Comment:
Since we change those lines, we should adapt them to the coding style:
context_mutex, context_instance_counter, etc.

Reply:
done